passes/source2il.nim

```diff
@@ -149,6 +149,12 @@ using
   expr: var IrNode
   stmts: var seq[IrNode]
 
+func align(val: SizeUnit, to: SizeUnit): SizeUnit =
+  ## Rounds `val` to the multiple of `to`, the latter which must be a power of
+  ## two.
```

Comment:
```suggestion
  ## Rounds `val` to the multiple of `to`, the latter must be a power of two.
```

Just playing a bit of word golf to get a one line comment

Reply:
Yea, the short-one-word-comments are a bit awkward.